packages/react/src/components/dropdown/select/Select.stories.tsx

```diff
@@ -96,7 +96,7 @@ export const Controlled = (args) => {
         style={{ marginTop: 'var(--spacing-s)' }}
       />
 
-      <Button onClick={() => setSelectedItems(null)} style={{ marginTop: 'var(--spacing-l)' }}>
+      <Button onClick={() => setSelectedItems([])} style={{ marginTop: 'var(--spacing-l)' }}>
```

Comment:
I got one question, why don't we keep this as `null` since the component will now accept `null` values?

Reply:
At the moment, only the single Select accepts null as values. Should the multi Select accept nulls as well?
I can fix it but I was not sure about it at first

Reply:
Ah, I didn't notice that this story is for the multi-select. Not sure if making it nullable will be needed in some scenarios 🤷    

But looks good to me :)